SelfContained/Program.cs

```diff
@@ -10,7 +11,23 @@ public static class Program
     [RequiresUnreferencedCode ("Calls Terminal.Gui.Application.Run<T>(Func<Exception, Boolean>, ConsoleDriver)")]
     private static void Main (string [] args)
     {
-        Application.Run<ExampleWindow> ().Dispose ();
+        Application.Init ();
+
+        if (Equals (Thread.CurrentThread.CurrentUICulture, CultureInfo.InvariantCulture))
```

Comment:
Does this need to be in the sample? Does EVERY self-contained-single app need these checks? If not, please remove.

Reply:
The sample code should only include code that is required to show developers how to do something.

Your response confuses me because you say "it's only needed to ensure it not breaks" and "this is isn't needed because it's only for single-file". But this IS a single-file sample.

Reply:
I was referring the localization not breaking anymore.

Reply:
I still don't understand. 

Can you please update `SelfContained/Program.cs` to have the bare-minimum code required to be a clear and complete sample for a single file self-contained TUI app?

Reply:
@tig is this solution https://github.com/gui-cs/Terminal.Gui/pull/3612/commits/ba4ab0c8c6aab183a704ab2016f6c765a27b9d5c enough?
Also the respective `SelfContained.csproj` is not compatible with the creation of a single-file self-contained application because in `Debug` mode it references the `Terminal.Gui` project and in `Release` mode it references the `Terminal.Gui` nuget package. My main reason for creating this project was to test and verify that `Terminal.Gui` complies with trimmed applications. Normally a user who wants a single-file application knows how to do it.